uint8 system_id
uint8 component_id
string timestamp
bool armed
bool manual_input
bool HIL_simulation
bool stabilized_mode
bool guided_mode
bool auto_mode
bool test_mode
bool custom_mode